addons/account_edi_ubl_cii/data/cii_22_templates.xml

```diff
@@ -61,18 +61,6 @@
                                 <ram:RateApplicablePercent t-esc="tax_detail_vals['amount']"/>
                             </ram:ApplicableTradeTax>
                         </t>
-
-                        <!-- Discount. -->
-                        <!-- A discount in percent can only be on the whole document (CalculationPercent), not on a line... -->
-                        <ram:SpecifiedTradeAllowanceCharge t-if="line.discount">
-                            <ram:ChargeIndicator>
-                                <udt:Indicator>false</udt:Indicator>
-                            </ram:ChargeIndicator>
-                            <ram:ActualAmount t-esc="format_monetary(line.price_unit * line.discount/100, 2)"/>
-                            <!-- https://unece.org/fileadmin/DAM/trade/untdid/d16b/tred/tred5189.htm -->
-                            <ram:ReasonCode>95</ram:ReasonCode>
-                        </ram:SpecifiedTradeAllowanceCharge>
-
```

Comment:
So what about this? You just remove it from the tests etc?
Is this linked to the computation of taxes?

Reply:
Yes, I remove it. At the moment, a discount is accounted twice in the xml: in AppliedTradeAllowanceCharge and in SpecifiedTradeAllowanceCharge. It's not correct. We have to choose to keep one or the other and I chose to keep the AppliedTradeAllowanceCharge (corresponding to a rebate: a difference between the gross unit price and the net unit price) because it's the one use in the legacy facturx module.
It's not linked to the computation of taxes but it is comprised in the price_unit computation.

Reply:
Nice tests :sweat_smile: